package.json

```diff
@@ -3,15 +3,14 @@
   "version": "0.0.1",
   "description": "Angular Sample Applications for Okta",
   "scripts": {
-    "test": "npm run test:e2e",
+    "test": ".\\scripts\\runTests.sh || npm run test:e2e",
```

Comment:
If this sh script achieves the same thing, is there not a cross-platform way to invoke it for both platforms?

Reply:
Actually, there is a slight difference between the 2.
In windows, we don't set the `TEST_TYPE=implicit` and `cross-env` doesn't work because it doesn't set the env var for child processes (invoked by `npm test`)
For now, we have to live with this, if it's not too bad.